joomla/joomla-cms
# [4.0] Installation languages model install articles with real null value for publish down time

Pull Request for Issue # .

### Summary of Changes

This is a follow up to recently merged PR's which changed datetime columns in database for real null values.

It makes sure that the `addArticle` routine in file `installation/src/Model/LanguagesModel.php` does not add any article with an old fashioned null value (which might be invalid on newer versions of MySQL databases depending on server and session settings) for any datetime column in database.

### Testing Instructions

Code review should be enoug: Check that in the file modified by this PR there is no other place where an old fashioned null datetime is written to database.

### Expected result

No old fashioned null datetime is written to database.

### Actual result

Old fashioned null datetime is written to database for the published down time of articles.

### Documentation Changes Required

None.
